exasol_transformers_extension/deployment/language_container_deployer.py

```diff
@@ -40,15 +54,37 @@ def _upload_container(self) -> PurePosixPath:
         logging.debug("Container is uploaded to bucketfs")
         return PurePosixPath(path_in_udf)
 
-    def _generate_alter_command(self, alter_type: str,
-                                path_in_udf: PurePosixPath) -> str:
+    def register_container(self, alter_type: LanguageRegLevel = LanguageRegLevel.Session,
+                           path_in_udf: Optional[PurePosixPath] = None) -> None:
+        """
+        Registers the SLC container at the required level.
+
+        alter_type  - Language registration level, defaults to the SESSION.
+        path_in_udf - If known, a path where the container is uploaded as it's seen by a UDF.
+        """
+        alter_command = self.generate_alter_command(alter_type, path_in_udf)
+        self._pyexasol_conn.execute(alter_command)
+        logging.debug(alter_command)
+
+    def generate_alter_command(self, alter_type: LanguageRegLevel,
+                               path_in_udf: Optional[PurePosixPath] = None) -> str:
+        """
+        Generates an SQL command to register the SLC container at the required level. The command will
+        preserve existing registrations of other containers identified by different language aliases.
+        Registration of a container with the same alias, if exists, will be overwritten.
```

Comment:
Do you want to have this option at both API and CLI levels? What should be the default?

Reply:
Yeah, I think, it makes sense to have it at both. I am inclined to throw an error when it already exists as the default for the standalone project. And, for the sandbox we would overwrite it always overwrite, because there is the situation different.